style.css

```diff
@@ -169,4 +171,139 @@ button:focus {
   transform: rotateY(180deg);
 }
 
+@media (max-width: 980px) {
+  .title {
+    font-size: 36px;
+  } 
+
+  .text {
+     font-size: 12px;     
+  }
+
+  .difficults {
+    font-size: 32px;
+    line-height: 34px;
+    padding-left: 119px;    
+  }
+
+  .card-wrap {
+    width: 38%;
+  }
+   
+  .start-card {
+    width: 100%; 
+  }
+
+  .difficult-check {
+    top: 36px;    
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/jsFinalTask/pullRequest?prid=6141138)